src/main/java/com/bravo/user/dao/repository/AddressRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.bravo.user.dao.repository;
+
+import com.bravo.user.dao.model.Address;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface AddressRepository extends JpaRepository<Address, String>, JpaSpecificationExecutor<Address> {
```

Comment:
`JpaSpecificationExecutor` is not really required here since we're only querying on a single input.

     List<Address> findByUserId(final String userId);

https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#repositories.core-concepts

Reply:
Oh yeah that's right. I'm rusty there it seems. Last time I had to add a method to a repository interface was in 2015, or 7 years ago